README.md

```diff
@@ -41,6 +41,9 @@ Please consider the experience with this plug-in as experimental until Neovim 0.
 You can find the current roadmap [here](https://github.com/nvim-treesitter/nvim-treesitter/projects/1).
 The roadmap and all features of this plugin are open to change, and any suggestion will be highly appreciated!**
 
+*If you want to use this plugin with Neovim 0.5, please use the
+0.5-compat branch*
+
```

Comment:
Maybe add a note that some improvements and bug fixes may require using neovim nightly?

Reply:
Done